### PR TITLE
Feat: [HL-70] Saved recipes table view cell 🍕.

### PR DESCRIPTION
**Pull request by ahmednasr8898**
#### Proposed Changes

Describe the changes proposed in this pull request.

1- Make UI for saved recipes table view cell.
2- Make saved recipes table view cell view model.
3- Adding Unit Test.

#### Related Issues

List any related issues or pull requests.

#### Additional Information

Provide any additional information or context that may be relevant.

#### Screenshot

Required for any UI changes

Screenshot 2023-06-21 at 2 45 41 PM

#### Checklist

- [ ] I have tested these changes locally.
- [ ] I have added appropriate documentation or updated existing documentation.
- [ ] I have added appropriate test coverage or updated existing test coverage.
- [ ] I have updated the changelog (if applicable).
- [x] I have followed the project's code style and formatting guidelines.
- [ ] I have reviewed and adhered to the project's contributing guidelines.

### CODE REVIEW

File: Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg
Comment (ahmdmhasn) [inline Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg:1]: [pattern-food.pdf](https://github.com/motoon-eg/healthy/files/11825991/pattern-food.pdf)

Please use this `pattern-food` image as a placeholder.

File: Healthy/Resources/Assets.xcassets/background-gradient.imageset/Contents.json
Comment (ahmdmhasn) [inline Healthy/Resources/Assets.xcassets/background-gradient.imageset/Contents.json:1]: Please create the gradient in code.

File: Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg
Comment (ahmdmhasn) [inline Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg:1]: Please use PDF or SVG.

Comment (ahmdmhasn) [reply Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg:1]: Also, I still don't recognize this image 😅

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        // TODO: Configure cell with view model.
+    }
+}
+
+// MARK: Private Handlers
+
+private extension SavedRecipesTableViewCell {
+    func configureLayout() {
+        contentView.layer.cornerRadius = 10
+        // TODO: Waiting for add style to title label.
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]: Please add the new styles in the same ticket 🙏🏼

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesCellViewModel/SavedRecipesCellViewModelType.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/// SavedRecipesCellViewModel Input & Output
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesCellViewModel/SavedRecipesCellViewModelType.swift]: Not needed.

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesCellViewModel/SavedRecipesCellViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import Foundation
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesCellViewModel/SavedRecipesCellViewModel.swift]: No needed

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+class SavedRecipesTableViewCell: UITableViewCell {
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]:
```suggestion
extension SavedRecipesTableViewCell { 
    struct ViewModel {
        // Here goes your implementation....
    }
}

final class SavedRecipesTableViewCell: UITableViewCell {
```

File: HealthyTests/Classes/Modules/SavedRecipes/Cell/TableViewMock.swift [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import UIKit
+@testable import Healthy
+
+final class SavedRecipesTableViewMock: UITableView, UITableViewDelegate, UITableViewDataSource {
```
Comment (ahmdmhasn) [inline HealthyTests/Classes/Modules/SavedRecipes/Cell/TableViewMock.swift]:
```suggestion
We can reuse it 👏🏼

Also, please move it to `HealthyTests/Mocks/UITableViewMock.swift
final class UITableViewMock: UITableView, UITableViewDelegate, UITableViewDataSource {
```

Comment (ahmdmhasn) [reply HealthyTests/Classes/Modules/SavedRecipes/Cell/TableViewMock.swift]: Also, you can inject the cell type

File: HealthyTests/Classes/Modules/SavedRecipes/Cell/TableViewMock.swift [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "SavedRecipesTableViewCell", for: indexPath) as! SavedRecipesTableViewCell
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 150
+    }
```
Comment (ahmdmhasn) [inline HealthyTests/Classes/Modules/SavedRecipes/Cell/TableViewMock.swift]:
```suggestion
```

File: HealthyTests/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCellTests.swift [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    override func setUp() {
+        sut = SavedRecipesTableViewCell()
+        mockTableView = SavedRecipesTableViewMock(frame: CGRect(x: 0, y: 0, width: 200, height: 400), style: .plain)
+        mockTableView.register(UINib.init(nibName: "SavedRecipesTableViewCell", bundle: nil),
+                               forCellReuseIdentifier: "SavedRecipesTableViewCell")
+    }
+    
+    // MARK: Tests
+    
+    func test_SavedRecipesCellIsFound() {
```
Comment (ahmdmhasn) [inline HealthyTests/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCellTests.swift:21]: Please don't fotget given - when - then pattern

File: HealthyTests/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCellTests.swift [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    
+    private var sut: SavedRecipesTableViewCell!
+    private var mockTableView: SavedRecipesTableViewMock!
+    
+    // MARK: Lifecycle
+    
+    override func setUp() {
+        sut = SavedRecipesTableViewCell()
+        mockTableView = SavedRecipesTableViewMock(frame: CGRect(x: 0, y: 0, width: 200, height: 400), style: .plain)
+        mockTableView.register(UINib.init(nibName: "SavedRecipesTableViewCell", bundle: nil),
```
Comment (ahmdmhasn) [inline HealthyTests/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCellTests.swift]: Please check how we register the collection view cells and implement a similar approach. Feel free to alugn with @abdallah-7698

**Review by ahmdmhasn — COMMENTED** [review]
Great PR, Thanks! 🎉❤️

File: Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg
Comment (ahmednasr8898) [reply Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg:1]: already use it.

**Review by esraakhaled — APPROVED** [review]

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    // MARK:  Lifecycle
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureLayout()
+    }
+    
+    // MARK:  Configure
+    
+    func configureCell(viewModel: ViewModel) {
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]:
```suggestion
    func update(with viewModel: ViewModel) {
```

File: Healthy/Classes/Extensions/UILabel+Style.swift [hunk 10]
```diff
@@ -14,10 +14,13 @@ extension UILabel {
         case signupSubtitle
         case textFieldTitleLabel
         case messageLabel
+        case savedRecipesTitle
+        case savedRecipesSubTitle
+        case time
```
Comment (ahmdmhasn) [inline Healthy/Classes/Extensions/UILabel+Style.swift]: Styles should be abstract and should not use any business names.

File: Healthy/Classes/Extensions/UILabel+Style.swift [hunk 11]
```diff
@@ -14,10 +14,13 @@ extension UILabel {
...
 
         var labelTextColor: UIColor {
             switch self {
-            case .splashHeaderLabel, .splashTitle, .splashSubtitle:
+            case .splashHeaderLabel, .splashTitle, .splashSubtitle, .savedRecipesTitle, .savedRecipesSubTitle, .time:
```
Comment (ahmdmhasn) [inline Healthy/Classes/Extensions/UILabel+Style.swift]:
```suggestion
            case .splashHeaderLabel, 
                     .splashTitle, 
                     .splashSubtitle, 
                     .savedRecipesTitle, 
                     .savedRecipesSubTitle, 
                     .time:
```

File: Healthy/Classes/Extensions/UILabel+Style.swift [hunk 12]
```diff
@@ -83,4 +90,6 @@ private enum LabelFont {
     static let signupSubtitle = UIFont.systemFont(ofSize: 11, weight: .regular)
     static let textFieldTitleLabel = UIFont.systemFont(ofSize: 14, weight: .regular)
     static let messageLabel = UIFont.systemFont(ofSize: 20, weight: .semibold)
+    static let savedRecipesTitleLabel = UIFont.systemFont(ofSize: 14, weight: .bold)
```
Comment (ahmdmhasn) [inline Healthy/Classes/Extensions/UILabel+Style.swift]: Fonts are integrated, please use them.

File: Healthy/Classes/Extensions/UITableView+RegisterNib.swift [hunk 13]
```diff
@@ -0,0 +1,27 @@
+import UIKit
```
Comment (ahmdmhasn) [inline Healthy/Classes/Extensions/UITableView+RegisterNib.swift:1]: Excellent

File: HealthyTests/Classes/Mocks/UITableViewMock.swift [hunk 14]
```diff
@@ -0,0 +1,16 @@
+import UIKit
+@testable import Healthy
+
+final class UITableViewMock: UITableView, UITableViewDelegate, UITableViewDataSource {
```
Comment (ahmdmhasn) [inline HealthyTests/Classes/Mocks/UITableViewMock.swift]: Please use the generic approach we've talked about.

File: Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg
Comment (ahmdmhasn) [reply Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg:1]: Not updated.

File: Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg
Comment (ahmdmhasn) [inline Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg:1]: Please use single style.

Comment (ahmdmhasn) [reply Healthy/Resources/Assets.xcassets/icon-stroke.imageset/Union@1x.svg:1]: Use a better name please.

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+    }
+}
+
+// MARK: View Model
+
+extension SavedRecipesTableViewCell {
+    struct ViewModel {
+        // Here goes your implementation....
+    }
+}
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]: Please move this block to the top of the file.

**Review by ahmdmhasn — CHANGES_REQUESTED** [review]

**Review by ahmdmhasn — COMMENTED** [review]
Please check the comments and let me know.

File: Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg
Comment (ahmednasr8898) [reply Healthy/Resources/Assets.xcassets/background-food.imageset/heart-healthy-food-1580231690.jpg:1]: i added it in cell Nib and when using kingfisher i will add it as placeholder.

**Comment by ahmednasr8898** [comment]
@ahmdmhasn Please can you check new changes.

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 16]
```diff
@@ -0,0 +1,50 @@
+import UIKit
+
+// MARK: View Model
+
+extension SavedRecipesTableViewCell {
+    struct ViewModel {
+        // Here goes your implementation....
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]: Please add the needed properties.

Comment (ahmdmhasn) [reply Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift]: Please don't forget onTap

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+    // MARK: Outlets
+    
+    @IBOutlet private(set) weak var recipeImageView: UIImageView!
+    @IBOutlet private(set) weak var titleLabel: UILabel!
+    @IBOutlet private(set) weak var chefNameLabel: UILabel!
+    @IBOutlet private(set) weak var rateView: UIView!
+    @IBOutlet private(set) weak var timeLabel: UILabel!
+    @IBOutlet private(set) weak var bookMarkView: UIView!
+    @IBOutlet private(set) weak var bookMarkImageView: UIImageView!
+
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift:27]:
```suggestion
    
    // MARK: Properties
    
    private var toggleBookmark: () -> Void = {}
```

File: Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.xib [hunk 18]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.xib:1]:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
    <device id="retina6_12" orientation="portrait" appearance="light"/>
    <dependencies>
        <deployment identifier="iOS"/>
        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21505"/>
        <capability name="Named colors" minToolsVersion="9.0"/>
        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
        <capability name="System colors in document resources" minToolsVersion="11.0"/>
        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
    </dependencies>
    <objects>
        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
        <tableViewCell contentMode="scaleToFill" selectionStyle="default" indentationWidth="10" rowHeight="196" id="KGk-i7-Jjw" customClass="SavedRecipesTableViewCell" customModule="Healthy" customModuleProvider="target">
            <rect key="frame" x="0.0" y="0.0" width="467" height="196"/>
            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
            <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="KGk-i7-Jjw" id="H2p-sc-9uM">
                <rect key="frame" x="0.0" y="0.0" width="467" height="196"/>
                <autoresizingMask key="autoresizingMask"/>
                <subviews>
                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="o1X-a1-R6g">
                        <rect key="frame" x="0.0" y="0.0" width="467" height="196"/>
                    </imageView>
                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="equalSpacing" translatesAutoresizingMaskIntoConstraints="NO" id="7rF-fF-ZKs">
                        <rect key="frame" x="8" y="8" width="451" height="180"/>
                        <subviews>
                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="equalCentering" alignment="bottom" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="5gZ-Q1-Az5">
                                <rect key="frame" x="0.0" y="0.0" width="451" height="16"/>
                                <subviews>
                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="hk2-NY-RsQ">
                                        <rect key="frame" x="415" y="0.0" width="36" height="16"/>
                                        <subviews>
                                            <stackView opaque="NO" contentMode="scaleToFill" spacing="3.5" translatesAutoresizingMaskIntoConstraints="NO" id="I0J-54-7OW">
                                                <rect key="frame" x="5.9999999999999982" y="4" width="24.333333333333329" height="8"/>
                                                <subviews>
                                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="AdF-ci-9sw">
                                                        <rect key="frame" x="0.0" y="0.0" width="8" height="8"/>
                                                        <color key="backgroundColor" name="rating"/>
                                                        <constraints>
                                                            <constraint firstAttribute="width" constant="8" id="2lS-Va-2pb"/>
                                                            <constraint firstAttribute="height" constant="8" id="8Ca-5s-5zF"/>
                                                        </constraints>
                                                    </view>
                                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="0.0" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Cxl-Ar-yq0">
                                                        <rect key="frame" x="11.666666666666686" y="0.0" width="12.666666666666664" height="8"/>
                                                        <fontDescription key="fontDescription" type="system" pointSize="8"/>
                                                        <nil key="textColor"/>
                                                        <nil key="highlightedColor"/>
                                                    </label>
                                                </subviews>
                                            </stackView>
                                        </subviews>
                                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                                        <constraints>
                                            <constraint firstItem="I0J-54-7OW" firstAttribute="centerY" secondItem="hk2-NY-RsQ" secondAttribute="centerY" id="4PE-d0-twI"/>
                                            <constraint firstAttribute="width" constant="36" id="XvX-j0-Fsc"/>
                                            <constraint firstAttribute="height" constant="16" id="gtz-rM-0Vu"/>
                                            <constraint firstItem="I0J-54-7OW" firstAttribute="centerX" secondItem="hk2-NY-RsQ" secondAttribute="centerX" id="zFr-Lv-kqu"/>
                                        </constraints>
                                    </view>
                                </subviews>
                            </stackView>
                            <stackView opaque="NO" contentMode="scaleToFill" alignment="bottom" spacing="17" translatesAutoresizingMaskIntoConstraints="NO" id="bgb-Ts-Q3m">
                                <rect key="frame" x="0.0" y="133.33333333333334" width="451" height="46.666666666666657"/>
                                <subviews>
                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="6" translatesAutoresizingMaskIntoConstraints="NO" id="lNe-2P-opI">
                                        <rect key="frame" x="0.0" y="0.0" width="334" height="46.666666666666664"/>
                                        <subviews>
                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="title" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="aYS-5J-z3w">
                                                <rect key="frame" x="0.0" y="0.0" width="334" height="20.333333333333332"/>
                                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                                <nil key="textColor"/>
                                                <nil key="highlightedColor"/>
                                            </label>
                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="chef name" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ckL-hc-CAD">
                                                <rect key="frame" x="0.0" y="26.333333333333314" width="334" height="20.333333333333329"/>
                                                <constraints>
                                                    <constraint firstAttribute="height" constant="20.329999999999998" id="mw5-Ep-tmb"/>
                                                </constraints>
                                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                                <nil key="textColor"/>
                                                <nil key="highlightedColor"/>
                                            </label>
                                        </subviews>
                                    </stackView>
                                    <stackView opaque="NO" contentMode="scaleToFill" alignment="center" spacing="16" translatesAutoresizingMaskIntoConstraints="NO" id="6uM-gO-hVr">
                                        <rect key="frame" x="351" y="22.666666666666657" width="100" height="24"/>
                                        <subviews>
                                            <stackView opaque="NO" contentMode="scaleToFill" distribution="fillProportionally" alignment="center" spacing="6" translatesAutoresizingMaskIntoConstraints="NO" id="oeN-wf-sOq">
                                                <rect key="frame" x="0.0" y="2" width="60" height="20.333333333333332"/>
                                                <subviews>
                                                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="clock" catalog="system" translatesAutoresizingMaskIntoConstraints="NO" id="Gm8-4H-PHh">
                                                        <rect key="frame" x="0.0" y="0.99999999999999822" width="20" height="18.666666666666664"/>
                                                        <color key="tintColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                                        <constraints>
                                                            <constraint firstAttribute="width" constant="20" id="kYH-WX-Htv"/>
                                                            <constraint firstAttribute="height" constant="20" id="sgj-lj-kTi"/>
                                                        </constraints>
                                                    </imageView>
                                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="time" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="AJO-fY-bBG">
                                                        <rect key="frame" x="26" y="0.0" width="34" height="20.333333333333332"/>
                                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                                        <nil key="textColor"/>
                                                        <nil key="highlightedColor"/>
                                                    </label>
                                                </subviews>
                                            </stackView>
                                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="qVl-MD-hVa">
                                                <rect key="frame" x="76" y="0.0" width="24" height="24"/>
                                                <subviews>
                                                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="icon-bookmark" translatesAutoresizingMaskIntoConstraints="NO" id="vjG-A2-KKw">
                                                        <rect key="frame" x="5" y="5" width="14" height="14"/>
                                                    </imageView>
                                                </subviews>
                                                <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                                                <constraints>
                                                    <constraint firstAttribute="trailing" secondItem="vjG-A2-KKw" secondAttribute="trailing" constant="5" id="18M-ot-bYy"/>
                                                    <constraint firstAttribute="height" constant="24" id="EtB-mI-ce4"/>
                                                    <constraint firstItem="vjG-A2-KKw" firstAttribute="top" secondItem="qVl-MD-hVa" secondAttribute="top" constant="5" id="MWI-NK-axN"/>
                                                    <constraint firstItem="vjG-A2-KKw" firstAttribute="leading" secondItem="qVl-MD-hVa" secondAttribute="leading" constant="5" id="Pm2-RS-XPV"/>
                                                    <constraint firstAttribute="bottom" secondItem="vjG-A2-KKw" secondAttribute="bottom" constant="5" id="xjL-oh-BBy"/>
                                                    <constraint firstAttribute="width" constant="24" id="xlK-uP-JbL"/>
                                                </constraints>
                                            </view>
                                        </subviews>
                                    </stackView>
                                </subviews>
                            </stackView>
                        </subviews>
                    </stackView>
                </subviews>
                <constraints>
                    <constraint firstItem="7rF-fF-ZKs" firstAttribute="leading" secondItem="H2p-sc-9uM" secondAttribute="leading" constant="8" id="51J-8L-joG"/>
                    <constraint firstAttribute="bottom" secondItem="7rF-fF-ZKs" secondAttribute="bottom" constant="8" id="CkL-Ki-yGo"/>
                    <constraint firstAttribute="trailing" secondItem="o1X-a1-R6g" secondAttribute="trailing" id="LSg-MH-T3g"/>
                    <constraint firstItem="7rF-fF-ZKs" firstAttribute="top" secondItem="H2p-sc-9uM" secondAttribute="top" constant="8" id="Lhb-Oa-Be3"/>
                    <constraint firstAttribute="trailing" secondItem="7rF-fF-ZKs" secondAttribute="trailing" constant="8" id="Q4Q-a2-NOv"/>
                    <constraint firstItem="o1X-a1-R6g" firstAttribute="leading" secondItem="H2p-sc-9uM" secondAttribute="leading" id="f41-4b-W1h"/>
                    <constraint firstAttribute="bottom" secondItem="o1X-a1-R6g" secondAttribute="bottom" id="mHv-62-WLi"/>
                    <constraint firstItem="o1X-a1-R6g" firstAttribute="top" secondItem="H2p-sc-9uM" secondAttribute="top" id="xW9-dH-Su5"/>
                </constraints>
            </tableViewCellContentView>
            <viewLayoutGuide key="safeArea" id="njF-e1-oar"/>
            <connections>
                <outlet property="bookMarkImageView" destination="vjG-A2-KKw" id="Ja4-Dh-b8f"/>
                <outlet property="bookMarkView" destination="qVl-MD-hVa" id="Yov-KK-Rl1"/>
                <outlet property="chefNameLabel" destination="ckL-hc-CAD" id="GZE-to-hPw"/>
                <outlet property="rateView" destination="hk2-NY-RsQ" id="Pqk-LQ-VJX"/>
                <outlet property="recipeImageView" destination="o1X-a1-R6g" id="RoM-3q-diG"/>
                <outlet property="timeLabel" destination="AJO-fY-bBG" id="nfJ-Yb-ViB"/>
                <outlet property="titleLabel" destination="aYS-5J-z3w" id="FEd-W6-TU7"/>
            </connections>
            <point key="canvasLocation" x="223.66412213740458" y="69.718309859154928"/>
        </tableViewCell>
    </objects>
    <resources>
        <image name="clock" catalog="system" width="128" height="123"/>
        <image name="icon-bookmark" width="12" height="14"/>
        <namedColor name="rating">
            <color red="1" green="0.67843137254901964" blue="0.18823529411764706" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
        </namedColor>
        <systemColor name="systemBackgroundColor">
            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
        </systemColor>
    </resources>
</document>
```

**Review by ahmdmhasn — APPROVED** [review]

**Review by ahmdmhasn — APPROVED** [review]

**Comment by ahmdmhasn** [comment]
🐑🐑🐑🚀🚀🚀

**Pull request merged**